runtime/src/bank.rs

```diff
@@ -4093,10 +4109,13 @@ impl Bank {
             self.rent_collector.rent.burn_percent = 50; // 50% rent burn
         }
 
-        if new_feature_activations.contains(&feature_set::inflation_kill_switch::id()) {
-            *self.inflation.write().unwrap() = Inflation::new_disabled();
-            self.fee_rate_governor.burn_percent = 100; // 100% fee burn
-            self.rent_collector.rent.burn_percent = 100; // 100% rent burn
+        if new_feature_activations.contains(&feature_set::full_inflation::id()) {
+            let mut full_inflation = Inflation::default();
```

Comment:
nits: I rather want to define `Inflation::full()`. well `ledger-tool` needs to access this for various simulation.